yarn.lock

```diff
@@ -5468,4 +5468,4 @@ zrender@5.3.2:
   resolved "https://registry.npmjs.org/zrender/-/zrender-5.3.2.tgz"
   integrity sha512-8IiYdfwHj2rx0UeIGZGGU4WEVSDEdeVCaIg/fomejg1Xu6OifAL1GVzIPHg2D+MyUkbNgPWji90t0a8IDk+39w==
   dependencies:
-    tslib "2.3.0"
+    tslib "2.3.0"
```

Comment:
Hi @pranavshuklaa, sorry for the late reply. Thank you for your contribution to our docs!

Did you edit the `yarn.lock` manually? An [EOL](https://prettier.io/docs/en/options.html#end-of-line) issue occurs ([LF vs. CRLF](https://stackoverflow.com/a/1552775/10369621)) and it is an issue related to tools such as your editor.

BTW, `yarn.lock` should always not be edited manually(what a time killer). it is only generated. If you keep using editor to change the EOL to `LF`, it is fine to me if the result could be no changes/diff lines in `yarn.lock`. But the best way is to `yarn install` and let `yarn` generate the lock file. May I know which version of `node`, `yarn`, `npm` you  are using? I want to know why dependencies should be modified to make the extension BOOT in your machine.

> Actually , the extension was working only after downloading these dependencies in my desktop.

Reply:
No worries @tyn1998 , All the best for you thesis ! We are rooting for you😀

For the question you asked. Yes i manually edited the yarn.lock since mine had some different set of dependencies .

>BTW, yarn.lock should always not be edited manually(what a time killer). it is only generated. If you keep using editor to change the EOL to LF, it is fine to me if the result could be no changes/diff lines in yarn.lock.
Ohh ! i will check it and make the necessary changes . 

> May I know which version of node, yarn, npm you are using? I want to know why dependencies should be modified to make the extension BOOT in your machine.

I have node v18.13.0 , yarn 1.22.19  , npm 9.6.4  installed.